node/pkg/driver/node.go

```diff
@@ -159,6 +159,18 @@ func (d *NodeService) NodeStageVolume(ctx context.Context, req *csi.NodeStageVol
 		logger.Debugf("NodeStageVolume Finished: multipath device [%s] is ready to be mounted by NodePublishVolume API.", mpathDevice)
 		return &csi.NodeStageVolumeResponse{}, nil
 	}
+	baseDevice := path.Base(mpathDevice)
+	rawSysDevices, err := d.NodeUtils.GetSysDevicesFromMpath(baseDevice)
```

Comment:
I know this is not part of the ticket, but ever since the [stage info file was removed](https://github.com/IBM/ibm-block-csi-driver/pull/282), `GetSysDevicesFromMpath` should have returned a list of strings, and not a "csv" string.
could you fix it while you're at it?

Reply:
done